### PR TITLE
Add GLTO + BEAST

### PR DESCRIPTION
**Pull request by Rizbe**

### CODE REVIEW

**Comment by editaahn** [comment]
Please note that images should be 256x256px in size.

**Comment by Rizbe** [comment]
@editaahn updated image to 256x256px

File: cosmos/injective.json [hunk 1]
```diff
@@ -70,6 +70,20 @@
       "coinDecimals": 6,
       "coinMinimalDenom": "factory/inj127l5a2wmkyvucxdlupqyac3y0v6wqfhq03ka64/qunt",
       "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/injective/factory/inj127l5a2wmkyvucxdlupqyac3y0v6wqfhq03ka64/QUNT.png"
+    },
+    {
+      "coinDenom": "GLTO",
+      "coinMinimalDenom": "peggy0xd73175f9eb15eee81745d367ae59309Ca2ceb5e2",
```
Comment (editaahn) [inline cosmos/injective.json:76]: Is it a CW token or factory token?

**Comment by Rizbe** [comment]
This is Peggy asset which is ETH asset bridged to INJ via their bridge.

File: cosmos/injective.json (hunk 1 above)
Comment (Rizbe) [reply cosmos/injective.json:76]: This is Peggy asset which is ETH asset bridged to INJ via their bridge.

**Comment by Rizbe** [comment]
@editaahn since this was reverted, do I need to do something to get this merged back in?

**Comment by editaahn** [comment]
I should have checked whether Keplr supports Peggy. I'll get back and let you know about it. Thanks

**Pull request merged**